gateway/src/services/config-manager-v2.ts

```diff
@@ -290,8 +290,17 @@ export class ConfigManagerV2 {
     // Rebase the file paths in config root if they're relative paths.
     for (const namespaceDefinition of Object.values(namespaceMap)) {
       for (const [key, filePath] of Object.entries(namespaceDefinition)) {
-        if (filePath.charAt(0) !== '/') {
-          namespaceDefinition[key] = path.join(configRootDir, filePath);
+        if (key === 'configurationPath') {
+          if (!path.isAbsolute(filePath)) {
+            namespaceDefinition[key] = path.join(configRootDir, filePath);
+          }
+        } else {
```

Comment:
Use an `else if` here. Just an `else` works but is difficult to read, and more error prone when there are more schema changes.